byte-counter/index.html

```diff
@@ -10,6 +10,7 @@
 <p><textarea autofocus>i ♥ u</textarea>
 <p>That’s <strong id=characters>5 characters</strong>, totaling <strong id=bytes>7 bytes</strong>. <a href=#%E2%82%AC id=permalink>#</a>
 <p>Pro tip: add <code>https://mothereff.in/byte-counter#<var>%s</var></code> to the custom search engines / location bar shortcuts in your browser of choice. Whenever I want to check string length / byte count, I just enter <code>len <var>some string</var></code> in my address bar.
+<p>CRLF Issues: If you notice any carriage return or line feed issues please note that browsers normalize the input in form elements by default. See <a href=http://www.whatwg.org/specs/web-apps/current-work/multipage/the-button-element.html#the-textarea-element>spec</a> for detailed explanation.</p>
```

Comment:
Updated link: https://html.spec.whatwg.org/multipage/forms.html#the-textarea-element